driver/modern_bpf/helpers/store/auxmap_store_params.h

```diff
@@ -787,18 +787,16 @@ static __always_inline void auxmap__store_socktuple_param(struct auxiliary_map *
 		 */
 		push__u8(auxmap->data, &auxmap->payload_pos, socket_family_to_scap(socket_family));
 		if(direction == OUTBOUND) {
-			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_remote);
+			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_peer);
 			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_local);
-			path = BPF_CORE_READ(socket_remote, addr, name[0].sun_path);
+			path = BPF_CORE_READ(socket_peer, addr, name[0].sun_path);  // empty
 		} else {
 			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_local);
-			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_remote);
+			push__u64(auxmap->data, &auxmap->payload_pos, (uint64_t)socket_peer);
 			path = BPF_CORE_READ(socket_local, addr, name[0].sun_path);
 		}
 
-		unsigned long start_reading_point;
-		char first_path_byte = *(char *)path;
-		if(first_path_byte == '\0') {
+		if(path[0] == '\0') {
```

Comment:
Avoid convoluted logic with multiple variables involved.

Reply:
i have the suspect it was used to avoid some verifier madness, but let's see :crossed_fingers: